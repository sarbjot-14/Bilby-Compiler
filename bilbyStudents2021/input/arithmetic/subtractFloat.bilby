main{
	print 3.2 - 5.7 $n;
	print 5.1 - 1.7 $n;
	
	
}