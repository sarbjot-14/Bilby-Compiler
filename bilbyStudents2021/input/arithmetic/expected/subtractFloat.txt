-2.500000
3.400000
